omod/src/main/webapp/pages/scheduleProviders.gsp

```diff
@@ -49,6 +48,11 @@
 ].flatten()
 ]) %>
 
+<script type="text/javascript">
+var includeWeekends = ('${includeWeekends}' === "true");
+</script>
+
+<%= ui.includeJavascript("appointmentschedulingui", "controllers/scheduleProvidersController.js") %>
```

Comment:
Sorry, I missed this earlier... why did you have to move this?

Reply:
I moved it down since it now depends on var "includeWeekends" which I introduced.

Reply:
Does it fail if you don't move it?  All the Javascript files should be included  at the top.  There are a couple other variables set this way (look at the javascript block in lines 53:64.

Reply:
Since angular module scheduleProvidersController.js depends on "includeWeekends"

Reply:
Let me try it out and see :)